async_timeout/__init__.py

```diff
@@ -56,13 +54,22 @@ def remaining(self):
             return None
 
     def _do_enter(self):
-        if self._timeout is not None:
-            self._task = current_task(self._loop)
-            if self._task is None:
-                raise RuntimeError('Timeout context manager should be used '
-                                   'inside a task')
-            self._cancel_at = self._loop.time() + self._timeout
-            self._cancel_handler = self._loop.call_at(self._cancel_at, self._cancel_task)
+        self._task = current_task(self._loop)
+        if self._task is None:
+            raise RuntimeError('Timeout context manager should be used '
+                               'inside a task')
+
+        if self._timeout is None:
```

Comment:
Move these lines before `self._task` check: it allows to run aiohttp client from tornado with disabled timeouts.